chia/wallet/wallet_node.py

```diff
@@ -324,19 +325,19 @@ async def _await_closed(self, shutting_down: bool = True):
             await asyncio.sleep(0.5)  # https://docs.aiohttp.org/en/stable/client_advanced.html#graceful-shutdown
         self.wallet_peers = None
 
-    def _set_state_changed_callback(self, callback: Callable):
+    def _set_state_changed_callback(self, callback: Callable[..., Any]) -> None:
```

Comment:
do we really not have typing for this callback?

Reply:
No, we don't. This is used in multiple places with function signatures that do not agree.

Seems out of scope for this PR.

Reply:
See https://github.com/Chia-Network/chia-blockchain/pull/14464